packages/components/src/components/progress-indicator/_progress-indicator.scss

```diff
@@ -81,6 +91,11 @@
     color: $link-01;
     box-shadow: 0 rem(3px) 0 0 $link-01;
   }
+  
+  .#{$prefix}--progress--space-equal .#{$prefix}--progress-label {
+    max-width: 100%;
+    padding-right: 0.75rem;
```

Comment:
We might need to switch this to `margin-right` to remove the extra underline, not sure if this will have any adverse effects though. 

<img width="163" alt="Screen Shot 2020-03-23 at 11 25 28 AM" src="https://user-images.githubusercontent.com/11928039/77350017-23b5e080-6cf9-11ea-847b-99d789b97097.png">

Reply:
done